### PR TITLE
Reflect new major version release.

### PR DESCRIPTION
**Pull request by balhar-jakub**

### CODE REVIEW

**Comment by balhar-jakub** [comment]
@nannanli It seems I can't even assign you as a reviewer. Can you assign yourselves?

File: download.md [hunk 1]
```diff
@@ -245,20 +245,25 @@ Zowe version 1.0.0 through 1.8.0 are only available as rollup convenience builds
...
-<p><img src="assets/img/zowe-lts.png" width="90%" /></p>
-<p><b>Version timeframe, Current, Active LTS, Maintenance</b></p>
+<p><img src="assets/img/major_releases.png" width="90%" /></p>
+<p><b>Version timeframe, Active , Maintenance</b></p>
 <ul>
-<li>New Zowe versions will enter <b>current</b> release status for six to nine months to allow consumers of Zowe to test, provide feedback, and adjust to any changes.</li>
-<li>After current release phase, Zowe will move to <b>Active LTS</b> status and will be deemed ready by the community for general use. <b>Active LTS</b> will have additional releases with both fixes and enhancements.</li>
-<li>Following a period of <b>Active LTS</b>, the Zowe version will enter <b>Maintenance</b> for fixes only.</li>
-<li>The combination of <b>Active LTS</b> and <b>Maintenance LTS</b> release is designated as <b>"long-term support”</b>, which provides two guarantees:
+<li>Zowe is released on regular schedule. Major version every two years. Minor versions on a schedule published in <a href="https://github.com/zowe/community/blob/master/Project%20Management/Schedule/Zowe%20PI%20%26%20Sprint%20Cadence.md">Community repository.</a></li>
```
Comment (nannanli) [inline download.md:251]:
```suggestion
<li>Zowe is released on regular schedule. A major version is released every two years and minor versions on a schedule published in the <a href="https://github.com/zowe/community/blob/master/Project%20Management/Schedule/Zowe%20PI%20%26%20Sprint%20Cadence.md">Zowe Community GitHub repository</a>.</li>
```

File: download.md [hunk 2]
```diff
@@ -245,20 +245,25 @@ Zowe version 1.0.0 through 1.8.0 are only available as rollup convenience builds
...
+<li>There are three different types of releases, released by Zowe community:
```
Comment (nannanli) [inline download.md:252]:
```suggestion
<li>The Zowe community provides three types of releases:
```

File: download.md [hunk 3]
```diff
@@ -245,20 +245,25 @@ Zowe version 1.0.0 through 1.8.0 are only available as rollup convenience builds
...
+  <ul>
+    <li style="list-style-type: disc">Maintenance - This release receives only security patches and fixes for the critical bugs. It's suitable for environment that prefers stability to new features.</li>        
+    <li style="list-style-type: disc">Active - This release receives updates regularly according to the schedule, and it is for the users that are willing to invest a bit more time into upgrades to get the latest features.</li>
```
Comment (nannanli) [inline download.md:255]:
```suggestion
    <li style="list-style-type: disc">Active - This release receives updates regularly according to the schedule.  It is suitable for users who are willing to invest a bit more time into upgrades to get the latest features.</li>
```

File: download.md [hunk 4]
```diff
@@ -245,20 +245,25 @@ Zowe version 1.0.0 through 1.8.0 are only available as rollup convenience builds
...
+    <li style="list-style-type: disc">Nightly - The Zowe publishes bleeding edge state of the project on the nightly basis. There are no guarantees around the stability of the build.</li>
+  </ul>
+</li>
+<li>Every major version will be for two years in the Active phase and then in the Maintenance phase for another two and half year. This brings to you the possibility to use only major versions in the maintenance mode</li>
```
Comment (nannanli) [inline download.md:259]:
```suggestion
<li>Every major version will be in the Active phase for two years and then transition into the Maintenance phase for another two and half year. This brings to you the possibility to use only major versions in the maintenance mode.</li>
```

**Review by nannanli — APPROVED** [review]
Hi @balhar-jakub , I just added some suggestions. Feel free to commit or reject the suggestions.

**Pull request merged**